docs/en/index.md

```diff
@@ -124,3 +124,36 @@ You can disable the native driver by this option (and the emulated will take con
 redis:
 	session: {native: off}
 ```
+
+## Key prefixes
+
+When you use two instances of one application with one Redis server, it is possible, that your data can be overwritten by second application instance.
+
+To avoid this problem you can define key prefixes in configuration, for example:
+
+Instance 1
+```yml
+redis:
+    keyPrefix: "instance1_"
+    host: 127.0.0.1
+    port: 6379
+    journal: on
+    session: on
+    storage: on
+    debugger: off
+```
+
+Instance 2
+```yml
+redis:
+    keyPrefix: "instance2_"
```

Comment:
Now that I think about it, someone might misinterpret this as adding the prefix to the client object - it would be a little more writing, but it would also make a lot more sense to me having them under the specific sections.

```yml
parameters:
	redis:
		namespace: instance1

redis:
	host: 127.0.0.1
	port: 6379
	journal:
		namespace: %redis.namespace%
	session:
		namespace: %redis.namespace%
	storage:
		namespace: %redis.namespace%
	debugger: off
```